engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -544,8 +548,19 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
     }
 
     TryUtil.sequence(executionDiffs) match {
-      case Success(diffs) if diffs.exists(_.containsNewEntry) => startRunnableScopes(data.mergeExecutionDiffs(diffs))
-      case Success(diffs) => data.mergeExecutionDiffs(diffs)
+      case Success(diffs) =>
+
+        val startingEvents = for {
```

Comment:
also discussed in person, comments as to the reasoning behind the two spots for publishing these events